src/components/inspectors/ElementDestination.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+  <div>
+    <b-form-group :label="$t('Element Destination')">
+      <b-form-select id="" v-model="destinationType" @change="destinationTypeChange" data-test="element-destination-type">
+        <b-form-select-option :value="null" disabled>{{ $t('Select element destination') }}</b-form-select-option>
```

Comment:
Missing translation

Reply:
added to https://github.com/ProcessMaker/processmaker/pull/6875/files